src/main/java/com/de_alone/dokkang/repository/UserRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.de_alone.dokkang.repository;
+
+import com.de_alone.dokkang.models.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
```

Comment:
(I am new to spring boot), but shouldn't we also include "@Component" here, because this method is "@Autowired" in the controller class

Reply:
I'm not sure because I'm also not good at spring, so I looked it up
-> If you attach @Component, the parent annotation of @Repository, it is created in the root container the same way, but it is not used well because it is less visible.

Reply:
Ah, I see, then nevermind. Thank you for looking it up!